cli/src/main/java/com/box/l10n/mojito/cli/command/extractioncheck/ExtractionCheckNotificationSenderGithub.java

```diff
@@ -13,6 +13,8 @@
 @Configurable
 public class ExtractionCheckNotificationSenderGithub extends ExtractionCheckNotificationSender {
 
+  static final String COMMENT_REGEX = ".*\\*\\*i18n source string checks failed\\*\\*.*";
```

Comment:
The extraction check notification message is configurable so this regex should be too